doc/source/release/2.0.0-notes.rst

```diff
@@ -276,6 +276,9 @@ Deprecations
 
 * ``np.compat`` has been deprecated, as Python 2 is no longer supported.
 
+* ``numpy.int8`` will no longer support conversion of out of bounds python integers to integer arrays. For example, conversion of 255 to int8 will not return -1.
+  ``np.array(value).astype(dtype)`` will give the desired result.
```

Comment:
It would be good to note int8 as an example, but this affects all integer types (at least all lower precision types).
We could mention `np.iinfo(dtype).max` also, but this is fine.

Thanks a lot for following up!

Reply:
Yes, good idea. I have added an example of the use of np.iinfo(dtype).

``numpy.iinfo(dtype)``can be used to check the machine limits for data types. For example, ``np.iinfo(np.uint16)`` returns min = 0 and max = 65535.